src/vscode-dts/vscode.d.ts

```diff
@@ -10418,6 +10418,18 @@ declare module 'vscode' {
 
 	}
 
+	/**
+	 * The event that is fired when there is a change in {@link TreeView.focus tree view's focus}
+	 */
+	export interface TreeViewFocusChangeEvent<T> {
+
+		/**
+		 * Focused element.
+		 */
+		readonly focus: T | undefined;
```

Comment:
All API changes need to go through our API [proposal process.](https://github.com/Microsoft/vscode/wiki/Extension-API-process), so this will need to go in a new proposal `.d.ts` file.

Reply:
Moved it to a separate file and added a proposal check 🙏